rust/bridge/shared/src/ffi/convert.rs

```diff
@@ -333,6 +333,7 @@ impl crate::support::Env for Env {
 /// Implementation of [`bridge_handle`](crate::support::bridge_handle) for FFI.
 macro_rules! ffi_bridge_handle {
     ( $typ:ty as false ) => {};
+    ( $typ:ty as false, clone = true ) => {};
```

Comment:
The `$(, clone = $_:tt)?` from the bottom case is probably the most general approach, or even `$(, $($_:tt)*)` ("any remaining tokens after a comma").